### ISSUE TITLE
add HCP pipelines

### ISSUE THREAD

**Issue by stebo85** [NeuroDesk/neurocontainers#63]
https://github.com/Washington-University/HCPpipelines

**Comment by stebo85** [comment]
Yes, we have a bidsapp for these pipelines: bidsapphcppipelines_4.3.0

**Comment by lucsanara** [comment]
I am trying to work with hcpasl pipeline but do not succeed. Should it be also installed?

**Comment by stebo85** [comment]
Dear @lucsanara - the hcp-asl pipeline is not yet available in Neurodesk, but I opened an issue for it now: https://github.com/NeuroDesk/neurocontainers/issues/848 . Do you want to try to contribute this pipeline to Neurodesk?

**Comment by lucsanara** [comment]
Sure! Not sure how it works to install it. I ran:
conda create -n hcpasl python=3.11 cython 'numpy>=1.25'
conda activate hcpasl
pip install git+https://github.com/physimals/hcp-asl.git 
It did not work when entering the command process_hcp_asl

**Comment by stebo85** [comment]
@lucsanara - it needs a few more things to make this work. We can try to have a look at this in the next weeks - would that work for you or do you need it urgently?

**Comment by stebo85** [comment]
Dear @lucsanara,

hcpasl is available in Neurodesk - can you test it out and let us know if it is working well?
<img width="610" height="292" alt="Image" src="https://github.com/user-attachments/assets/b178e7ac-75a4-4647-9a09-4a749fa524d3" />
